src/main/sensors/voltage.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define VBAT_SCALE_DEFAULT 110
+#define VBAT_RESDIVVAL_DEFAULT 10
+#define VBAT_RESDIVMULTIPLIER_DEFAULT 1
+#define VBAT_SCALE_MIN 0
+#define VBAT_SCALE_MAX 255
+
+#define VBATT_LPF_FREQ  1.0f
+
+#ifndef MAX_VOLTAGE_METERS
+#define MAX_VOLTAGE_METERS 1 // VBAT - some boards have external, 12V and 5V meters.
+#endif
+
+typedef struct voltageMeterConfig_s {
+    uint8_t vbatscale;                      // adjust this to match battery voltage to reported value
```

Comment:
It would be great to get rid of this nonsense - three values to describe single floating point value.
Either use single fixed point value or (if absolutely necessary) numerator and denominator